src/components/Trade/types.ts

```diff
@@ -53,6 +53,7 @@ export type TradeState<C extends ChainId> = {
   amount: string
   receiveAddress: string | null // TODO: Implement
   slippage: number
+  isSendMax: boolean
```

Comment:
Yet another field that really belongs in context, rather than the form state.

To fix in a future refactor.